tests/dispatch_io.c

```diff
@@ -575,6 +581,12 @@ test_io_from_io(void) // rdar://problem/8388909
 		test_errno("chflags", errno, 0);
 		test_stop();
 	}
+#else
```

Comment:
I never realized that `#ifdef __APPLE__` it should be `#ifdef UF_IMMUTABLE` instead, other BSD variants have it too